lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -1472,7 +1517,7 @@ void GrandCentralPass::runOnOperation() {
     sort();
     llvm::dbgs() << "leafMap:\n";
     for (auto id : ids) {
-      auto fieldRef = leafMap.lookup(id);
+      auto fieldRef = leafMap.lookup(id).field;
       auto value = fieldRef.getValue();
       auto fieldID = fieldRef.getValue();
```

Comment:
Not related to this PR, but seems like it should be `fieldID = fieldRef.getFieldID()` , but it is only used for debugging.